google-account-plugin/src/com/google/cloud/tools/intellij/AccountPluginInitializationComponent.java

```diff
@@ -53,9 +56,26 @@ public void initComponent() {
     }
     if (!ApplicationManager.getApplication().isUnitTestMode()) {
       configureUsageTracking();
+      initUsageTracker();
     }
   }
 
+  @VisibleForTesting
+  void initUsageTracker() {
+    String platformPrefix = System.getProperty("idea.platform.prefix");
+    String trackingId = new PropertiesFilePluginFlags().getAnalyticsId();
+
+    Predicate<Object> userOptedIn = new Predicate<Object>() {
+      @Override
+      public boolean apply(@javax.annotation.Nullable Object object) {
+        return UsageTrackerManager.getInstance().hasUserOptedIn();
+      }
+    };
+
+    IdeaUsageTracker usageTracker = ServiceManager.getService(IdeaUsageTracker.class);
+    usageTracker.init(platformPrefix, trackingId, userOptedIn);
```

Comment:
We create and init a single instance of this class. The previous `GoogleUsageTracker` does the same thing: checking the opt-in status every time it tries to send a ping. I think it makes sense, because a user can opt in at an arbitrary point when using the IDE.